cmd/keymasterd/config.go

```diff
@@ -194,6 +201,11 @@ type DenyKeyConfig struct {
 	KeyDenyFPsshSha256 []string `yaml:"key_deny_list_ssh_sha256"`
 }
 
+type ExternalSignerConfig struct {
+	Type     string `yaml:"type"` // AWS|yubipiv
+	Location string `yaml:"location"`
```

Comment:
So, instead of having a single ARN/URL that gets parsed, the intent is to have `Location` parsed differently based on the `Type`?

Reply:
Correct, I moved the parsing so a separate function so that it becomes clearer.